Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN mvn clean install -T1C -U -f /src/dhis-2/dhis-web/pom.xml -DskipTests
 # Serve the packaged .war file (Tomcat)
 # Use a second build stage in a fresh container so we don't bloat it with the Maven build tools
 ##########
-FROM tomcat:8.5.34-jre8-alpine as serve
+FROM tomcat:8.5.46-jdk8-openjdk-slim as serve
```

Comment:
Perhaps we shouldn't do this in this PR but I'd like us to use the `tomcat:8.5-jdk-openjdk-slim` image to make sure that when they update Tomcat 8.5.x we get those fixes built-in without having to manually bump.

cc: @vilkg @Philip-Larsen-Donnelly